Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -614,41 +612,42 @@ TimeBeforeAfterRegex: !nestedRegex
 DateNumberConnectorRegex: !simpleRegex
   def: ^\s*(?<connector>\s+op)\s*$
 DecadeRegex: !simpleRegex
-  def: (?<decade>tweeduizend|21e eeuw|eenentwintigste eeuw)
+  def: (?<decade>(de\s+jaren\s+(vijftig|zestig|zeventig|tachtig|negentig))|(fifties|sixties|seventies|eighties|nineties|zeroes|tens|tweeduizend|21e eeuw|eenentwintigste eeuw))
 DecadeWithCenturyRegex: !nestedRegex
-  def: (de\s+)?(((?<century>\d|1\d|2\d)?(')?(?<decade>\d0)(')?s)|(({CenturyRegex}(\s+|-)(and\s+)?)?{DecadeRegex})|({CenturyRegex}(\s+|-)(and\s+)?(?<decade>tens|hundreds)))
+  def: (de\s+)?(((?<century>\d|1\d|2\d)?(')?(?<decade>\d0)(')?s)|(({CenturyRegex}(\s+|-)(en\s+)?)?{DecadeRegex})|({CenturyRegex}(\s+|-)(en\s+)?(?<decade>tien|honderd)))
   references: [ CenturyRegex, DecadeRegex ]
 RelativeDecadeRegex: !nestedRegex
-  def: \b((de\s+)?{RelativeRegex}\s+((?<number>[\w,]+)\s+)?decades?)\b
+  def: \b((de\s+)?{RelativeRegex}\s+((?<number>[\w,]+)\s+)?decennia?)\b
   references: [ RelativeRegex ]
 SuffixAfterRegex: !simpleRegex
-  def: \b((or|and)\s+(above|after|later|greater)(?!\s+than))\b
+  def: \b(((bij)\s)?(of|en)\s+(boven|na|later|groter)(?!\s+dan))\b
+DateAfterRegex: !simpleRegex
+  def: \b((of|en)\s+(hoger|later|groter)(?!\s+dan))\b
 YearPeriodRegex: !nestedRegex
-  def: ((((from|during|in)\s+)?{YearRegex}\s*({TillRegex})\s*{YearRegex})|(((between)\s+){YearRegex}\s*({RangeConnectorRegex})\s*{YearRegex}))
+  def: ((((vanaf|tijdens|gedurende|in)\s+)?{YearRegex}\s*({TillRegex})\s*{YearRegex})|(((tussen)\s+){YearRegex}\s*({RangeConnectorRegex})\s*{YearRegex}))
   references: [ YearRegex, TillRegex, RangeConnectorRegex ]
 ComplexDatePeriodRegex: !nestedRegex
-  def: (((from|during|in)\s+)?(?<start>.+)\s*({TillRegex})\s*(?<end>.+)|((between)\s+)(?<start>.+)\s*({RangeConnectorRegex})\s*(?<end>.+))
+  def: (((vanaf|tijdens|gedurende|in)\s+)?(?<start>.+)\s*({TillRegex})\s*(?<end>.+)|((tussen)\s+)(?<start>.+)\s*({RangeConnectorRegex})\s*(?<end>.+))
   references: [ YearRegex, TillRegex, RangeConnectorRegex ]
 UnitMap: !dictionary
   types: [ string, string ]
   entries:
-    millennium: 1000Y
-    eeuw: 100Y
-    centennium: 100Y
-    decennium: 10Y
-    decennia: 10Y
-    decenniën: 10Y
-    jaren: Y
-    jaar: Y
-    maanden: MON
-    maand: MON
+    millennium: 1000J
+    eeuw: 100J
+    centennium: 100J
+    decennium: 10J
+    decennia: 10J
+    jaren: J
+    jaar: J
+    maanden: MND
```

Comment:
Same `MON` to `MND`. As far as I know, those short term variants are for internal use only and shouldn't be translated.

Reply:
Resolved.